tvtk/pyface/tvtk_scene.py

```diff
@@ -683,6 +684,10 @@ def save_gl2ps(self, file_name, exp=None):
                 self._exporter_write(ex)
             else:
                 ex.write()
+            # Work around for a bug in VTK where it saves the file as a
+            # .pdf.gz when the file is really a PDF file.
```

Comment:
I looked at the sources and it seems the GL2PS code has changed considerably with VTK 7.0 The GL2PS exporter has been completely changed and looks like the bug shouldn't be there -- but I did not test as I do not have a VTK 7 build with GL2PS enabled on my system.